x-pack/plugin/esql/src/main/antlr/EsqlBaseLexer.g4

```diff
@@ -678,8 +677,8 @@ INSIST_MULTILINE_COMMENT : MULTILINE_COMMENT -> channel(HIDDEN);
 //
 mode FORK_MODE;
 FORK_LP : LP -> type(LP), pushMode(DEFAULT_MODE);
-FORK_RP : RP -> type(RP), popMode;
```

Comment:
The popup mode should already be completed by the sub fork, otherwise the mode stack is not completed, causing non-first subqueries to fail (since parsing is executed in a different mode).